support/docker/dev/janitor.json

```diff
@@ -19,13 +19,17 @@
       "proxy": "https"
     },
     "9000": {
-      "label": "PeerTube",
+      "label": "PeerTube API",
+      "proxy": "https"
+    },
+    "3000": {
+      "label": "PeerTube front",
       "proxy": "https",
       "preview": true
     }
   },
   "scripts": {
-    "Start PeerTube": "npm start",
+    "Start PeerTube": "npm run dev",
```

Comment:
Interesting, why did you need to switch this? What's the difference between `npm start` and `npm run dev`?

Reply:
`npm start` just serves start the built server and serves compiled client files.

`npm run dev` builds the server, run it on port 9000, builds the client on the fly and serves it on port 3000